### PR TITLE
init project

### PR DESCRIPTION
**Pull request by tranghoangthu812102**
ready

### CODE REVIEW

**Comment by tranghoangthu812102** [comment]
ready

**Comment by nxvinh222** [comment]
ready

**Comment by maihuysl1999** [comment]
Ok

**Comment by nxvinh222** [comment]
ok

**Comment by nxvinh222** [comment]
Ok

**Comment by tranghoangthu812102** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+gem "jbuilder", "~> 2.7"
+gem "mysql2", "~> 0.5.3"
+gem "puma", "~> 5.0"
+gem "rails", "~> 6.1.4"
+gem "sass-rails", ">= 6"
+gem "turbolinks", "~> 5"
+gem "webpacker", "~> 5.0"
+
+group :development, :test do
+  gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
```
Comment (ngocvt-0484) [inline Gemfile:15]: dùng gem pry-rails nhé e

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RubyNaiitei16LibraryManager</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (ngocvt-0484) [inline app/views/layouts/application.html.erb:9]: nháy "

**Comment by tranghoangthu812102** [comment]
ready

**Comment by tranghoangthu812102** [comment]
ready

**Pull request merged**